changelog/_unreleased/2025-03-27-introduce-snippets-for-theme-variables.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
as we want to get rid of those single changelog files in the near future, it would be nice, if you could directly update the upgrade guide files. 

so introduction of deprecations in the current 6.7 guide and the removal in 6.8 (e.g)

Reply:
Okay 👍

Reply:
@mitelg Is this documented somewhere as the new approach? I was under the impression that the UPGRADE.md files are meant to describe what needs to be changed to be compatible with the next major version. Adding deprecations to the 6.7 upgrade guide seems a bit confusing to me, as those changes aren’t required immediately. Just trying to understand the reasoning here – happy to follow the new structure if that’s the direction.